### ISSUE TITLE
Run an Image classification pipeline and test development time

### ISSUE THREAD

**Issue by rubix-coder** [rubix-coder/github-copilot#1]
- [ ] base lib : OpenCV
- [ ] dataset : COCO
- [ ] model : Custom Neural network
- [ ] loss fn.: cross_entropy
- [ ] optimiser : ADAM
- [ ] class: (human, bike, dog, car)
